entities/entities/darkrp_laws/cl_init.lua

```diff
@@ -33,26 +33,21 @@ end
 local function addLaw(inLaw)
     local law = DarkRP.textWrap(inLaw, "TargetID", 522)
 
-    Laws[#Laws + 1] = law
+    table.insert( Laws, law )
+    hook.Run("addLaw", #Laws + 1, law)
```

Comment:
Since this is now done after the table insertion, the 2nd parameter is now wrong. The `+ 1` should be removed.